sim/hunter/hunters_mark.go

```diff
@@ -0,0 +1,28 @@
+package hunter
+
+import (
+	"github.com/wowsims/cata/sim/core"
+)
+
+func (hunter *Hunter) registerHuntersMarkSpell() {
+	hunter.RegisterSpell(core.SpellConfig{
+		ActionID: core.ActionID{SpellID: 1130},
+		ProcMask: core.ProcMaskEmpty,
+		Flags:    core.SpellFlagAPL,
+
+		FocusCost: core.FocusCostOptions{
+			Cost: 0,
+		},
+		Cast: core.CastConfig{
+			DefaultCast: core.Cast{
+				GCD: core.GCDDefault,
+			},
+			IgnoreHaste: true,
+		},
+
+		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+			hm := core.HuntersMarkAura(target)
+			hm.Activate(sim)
```

Comment:
You are creating a new aura everytime you cast this spell.

What you shoould do is create an enemy aura array in the registration then in the cast activate the proper aura based on your target. 

If hunters mark can be active only on 1 target at the same time you still want to support to be able to activate it on any of the targets you have and make sure to deactivate it for the old target which can be done by just deactivating them all before activating it for the target (check NewEnemyAuraArray and how we use it around other debuffs)